routes/originComposer.js

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation and others. Licensed under the MIT license.
+// SPDX-License-Identifier: MIT
+
+const asyncMiddleware = require('../middleware/asyncMiddleware')
+const router = require('express').Router()
+const requestPromise = require('request-promise-native')
+const { uniq } = require('lodash')
+
+router.get(
+  '/:namespace?/:name/revisions',
+  asyncMiddleware(async (request, response) => {
+    const { name } = request.params
+    const url = `https://packagist.org/packages/${name}.json`
+    const answer = await requestPromise({ url, method: 'GET', json: true })
+    const result = answer.package.versions
```

Comment:
Should it be packages instead of package?

Reply:
The returned versions seem to be unsorted. Could you please sort them similarly to originNpm.js?

Reply:
Its package because this is a more specific api endpoint https://packagist.org/apidoc#get-package-data

GET https://packagist.org/packages/[vendor]/[package].json has more package information and takes us straight to package details instead of having to search for package from list of packages returned.